assets/javascripts/kitten/components/box/link-box/index.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { ArrowIcon } from '../../../components/icons/arrow-icon'
+import { Text } from '../../../components/typography/text'
+import { Marger } from '../../../components/layout/marger'
+import COLORS from '../../../constants/colors-config'
+import { ScreenConfig } from '../../../constants/screen-config'
+import { pxToRem } from '../../../helpers/utils/typography'
+
+const Link = styled.a`
+  display: flex;
+  color: ${COLORS.font1};
+  text-decoration: none;
+`
+
+const ParagraphMarger = styled(Marger)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-grow: 1;
+  padding-left: ${pxToRem(20)};
+
+  @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+    padding-left: ${pxToRem(30)};
+  }
+`
+
+const Icon = styled.div`
+  display: none;
+  margin-top: ${pxToRem(-2)};
+  margin-left: ${pxToRem(-2)};
+  margin-bottom: ${pxToRem(-2)};
+  align-items: center;
+  justify-content: center;
+  min-width: ${pxToRem(90)};
+  background-color: ${COLORS.primary4};
+
+  @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+    display: flex;
+  }
+`
+
+const Arrow = styled(props => (
+  <div {...props}>
+    <ArrowIcon className="k-ButtonIcon__svg" />
+  </div>
+))`
+  display: flex;
+  align-items: center;
+  padding: ${pxToRem(15)} ${pxToRem(22)} ${pxToRem(15)} ${pxToRem(18)};
+  transition: all 0.4s ease-in-out;
+
+  @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+    padding-left: ${pxToRem(30)};
+    padding-right: ${pxToRem(32)};
+  }
+`
+
+const Container = styled.div`
+  display: flex;
+  min-height: ${pxToRem(90)};
+  width: 100%;
+  box-sizing: border-box;
+  color: ${COLORS.font1};
+  background-color: ${COLORS.background1};
+  border: ${pxToRem(2)} solid ${COLORS.line1};
+
+  :hover {
+    background-color: ${COLORS.background2};
+
+    ${Arrow} {
+      transform: translate(${pxToRem(5)}, ${pxToRem(0)});
+    }
+  }
+
+  :active {
```

Comment:
est-ce qu'il ne vaut mieux pas ajouter un &, pour s'assurer qu'il s'agit bien de l'élément quand il est actif, et non un élément enfant actif ?
 
```suggestion
  &:active {
```

Reply:
Oui t'as raison. Sans le `&` ca fonctionne car il l'attache à la classe principale du composant mais du coup c'est un peu magique de `styled-components`. Je vais l'ajouter pour éviter qu'on ait à se poser la question.